snap/hooks/post-refresh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+set -eu
+
+PREVIOUS_VERSION=$(snapctl get previous-version)
+INITIAL_ALLOWED_USERS_VERSION="0.2.0"
+
+version_less_than() {
+  [ "$(printf '%s\n' "${@}" | sort -V | head -n1)" = "$1" ]
+}
+
+echo_and_log_to_journal() {
+  echo "${@}"
+  logger -t $SNAP_NAME "${@}"
```

Comment:
did you test that the messages actually end up in the journal (i.e. that the snap sandbox lets us print to the host's journal)?

Reply:
yes, do you get a different behavior?

Reply:
I did not test it

Reply:
Ok, I thought I was seeing ghosts :P